lib/config/flat-config-schema.js

```diff
@@ -262,6 +262,20 @@ const booleanSchema = {
     validate: "boolean"
 };
 
+/** @type {ObjectPropertySchema} */
+const severitySchema = {
+    merge(first, second) {
+        const value = second === void 0 ? first : second;
+
+        return normalizeSeverity(value);
+    },
+    validate(value) {
+        if (!(["true", "false", "error", "warn", "off", 2, 1, 0].includes(value) || typeof value === "boolean")) {
```

Comment:
String values `"true"` and `"false"` shouldn't be allowed.

Reply:
Removed.